Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -1122,6 +1124,15 @@ public void RemoveDevice(InputDevice device, bool keepOnListOfAvailableDevices =
             // Remove state monitors while device index is still valid.
             RemoveStateChangeMonitors(device);
 
+            device.removing = true;
+            if (!device.remote)
```

Comment:
Why exclude remote devices?

Reply:
Should remote devices have a valid device.currentStatePtr? At least in one of our tests, they did not.

Reply:
>At least in one of our tests, they did not.

That is curious. Sounds like a potential bug. From the perspective of the input system, a remote device looks (or should look) identical to a local one except that one flag.